api-library/api.yaml

```diff
@@ -370,8 +549,60 @@ components:
         status_message: "Available for chat"
         online_status: "online"
         created_at: "2023-01-01"
-        updated_at: "2023-01-02"
+        updated_at: "2023-01-02"    
+    UserAuth: 
+      type: object 
+      properties:
+        username: 
+          type: string 
+        email: 
+          type: string 
+          format: email 
+        
+        password: 
+          type: string 
+          format: password
+          
+      required: 
+        - username
+        - password
+        - email 
+        
+      example: 
+        username: "john_doe"
+        email: "john.doe@example.com"
+        password: "hashed_password_here"
 
+    UsersForUsers:
+      type: object 
+      properties:
+        id:
+          type: string
+          format: uuid
+        username:
+          type: string
+        email:
+          type: string
+          format: email
+        role_id:
+          type: string
+          format: uuid
+        bio:
+          type: string
+        avatar_url:
+          type: string
+          format: uri
+        status_message:
+          type: string
+        online_status:
+          type: string
+          enum:
+            - online
+            - away
+            - offline
+        created_at:
+          type: string
+          format: date
```

Comment:
I don't get why you defined UserForUsers schema

Reply:
ack.